packages/bson-bench/test/utils.ts

```diff
@@ -1,18 +1,22 @@
 import * as fs from 'fs/promises';
 import * as path from 'path';
 
+import { exists } from '../src/utils';
+
```

Comment:
dupe import?

Reply:
We're exporting `exists` on line 6 so that we only import utilities from this file in the other test files.